security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,26 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1-14"]
+== {cert-manager-operator} 1.14
+
+Issued: 2024-07-03
+
+The following advisory is available for the {cert-manager-operator} 1.14:
+
+* link:https://errata.engineering.redhat.com/advisory/128547[RHEA-2024:128547-02]
+
+[id="cert-manager-operator-new-features-1-14"]
+=== New features and enhancements
+
+*FIPS compliant support*
+
+With this release, FIPS mode is now automatically enabled for {cert-manager-operator}. When installed on an {product-title} cluster in FIPS mode, {cert-manager-operator} ensures compatibility without affecting the cluster's FIPS support status.
+
+*NCM issuer*
+
+The {cert-manager-operator} for {product-title} now supports the NCM issuer. The `ncm-issuer` is a Kubernetes controller that integrates with the Nokia NetGuard Certificate Manager (NCM) PKI system to sign certificate requests. This integration streamlines the process of obtaining non-self-signed certificates for applications, ensuring their validity and keeping them up-to-date.
```

Comment:
```suggestion
The {cert-manager-operator} for {product-title} now supports the NCM issuer. The `ncm-issuer` is a cert-manager External Issuer that integrates with the Nokia NetGuard Certificate Manager (NCM) PKI system using a Kubernetes controller to sign certificate requests. This integration streamlines the process of obtaining non-self-signed certificates for applications, ensuring their validity and keeping them up-to-date.
```

Reply:
Cc: @lunarwhite please help suggest in case you've a better statement for this, thanks!

Reply:
I'd suggest removing `for {product-title}` in the first sentence.

render will change from
```
The cert-manager Operator for Red Hat OpenShift for OpenShift Container Platform now supports the NCM issuer.
```
to
```
The cert-manager Operator for Red Hat OpenShiftnow supports the NCM issuer.
```

Reply:
How about adding the reference links
- https://www.nokia.com/networks/security-portfolio/netguard/certificate-manager/ to "NetGuard Certificate Manager"
- https://github.com/nokia/ncm-issuer to "ncm-issuer"

Reply:
@swghosh need your confirmation: do we need to also append the NCM issuer to "[Supported issuer types](https://76692--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/cert_manager_operator/#cert-manager-issuer-types_cert-manager-operator-about)" list?

Reply:
It is added through another PR. I think in release note, it is not needed.